Add compatibility section (#917)
github/view_component: docs/compatibility.md

```erb
<%= render_component Component.new(message: "bar") %>
```

## Bridgetown

> [Bridgetown](https://www.bridgetownrb.com/) added ViewComponent support as an **experimental**
> feature. It works via the [bridgetown-view-component gem](https://github.com/bridgetownrb/bridgetown-view-component).
> More information available [here](https://beta.bridgetownrb.com/docs/components/ruby#need-compatibility-with-rails-try-viewcomponent-experimental).

Reviewer: ```suggestion
[Bridgetown](https://www.bridgetownrb.com/) has **experimental** support for ViewComponent, via the [bridgetown-view-component gem](https://github.com/bridgetownrb/bridgetown-view-component). More information available [here](https://beta.bridgetownrb.com/docs/components/ruby#need-compatibility-with-rails-try-viewcomponent-experimental).
```
